src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard movie={movie} />
+      </div>
```

Comment:
move this cover inside component
```suggestion
    <MovieCard movie={movie} key={movie.imdbId} />
```